classes/controller/FrontController.php

```diff
@@ -495,6 +495,7 @@ protected function assignGeneralPurposeVariables()
             'time' => time(),
             'static_token' => Tools::getToken(false),
             'token' => Tools::getToken(),
+            'isDebugMode' => _PS_MODE_DEV_,
```

Comment:
Use snake case for consistency.

Alternative variable name:
```suggestion
            'debug' => _PS_MODE_DEV_,
```